aten/src/ATen/CPUApplyUtils.h

```diff
@@ -160,13 +161,13 @@ inline int64_t _max_dim_tensors(ArrayRef<Tensor> tensors) {
   return dim;
 }
 
-inline void iterate(){};
+inline void iterate(int64_t size){};
```

Comment:
nit: can we call this `stride`? It's effectively how far do you move in a single jump.

Reply:
I think it's easy to confuse with stride in the context of a Tensor. This is pretty much how big of a chunk you're processing at once, whereas the jumping happens outside the function.